### ISSUE TITLE
Vivaldi: "f" highlights links by providing shortcuts, but shortcuts can not be accessed

### ISSUE THREAD

**Issue by ThorirMarJonsson** [philc/vimium#2659]
`Vimium + Visual mode 1.44`

```Vivaldi | 1.11.917.43 (Stable channel) (32-bit)
-- | --
Revision | 6171cfeb79410505bfdd70a1434a4587da29614e-
OS | Windows
JavaScript | V8 6.0.286.54
Flash | (Disabled)
User Agent | Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.105 Safari/537.36 Vivaldi/1.92.917.43
Command Line | "C:\Users\user\AppData\Local\Vivaldi\Application\vivaldi.exe" --always-authorize-plugins --enable-blink-features=ResizeObserver --flag-switches-begin --flag-switches-end https://vivaldi.com/newfeatures?hl=en-us&version=1.11.917.43&os=W10.0.14393
Executable Path | C:\Users\user\AppData\Local\Vivaldi\Application\vivaldi.exe
Profile Path | C:\Users\user\AppData\Local\Vivaldi\User Data\Default
Compiler | MSVC 2015
```

The "f" key correctly highlights links by adding shortcut keys to them, but these are not accessible in Vivaldi. Pressing Escape removes the shortcuts from the screen so keyboard presses are read by Vimium, it's just typing in the link-shortcuts that does not work.

**Comment by mrmr1993** [comment]
The current version of Vimium is `1.60.3`, please try again using that.

**Comment by ThorirMarJonsson** [comment]
Thanks! This fixed the issue. 

For some reason when I went to the chrome store it looked like version 1.44 was the latest one leading me to think I was already at the latest version. After reading your comment, I tried reinstalling it regardless and then the displayed version in the store changed to version 1.60. Some glitch in the store I suppose. Anyway, Vimium is working perfectly again so I'm closing the issue.
